microsoft-365/security/mtp/mtp-enable.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Turn on Microsoft Threat Protection in the Microsoft 365 security center
-description: Learn how to enable Microsoft Threat Protection and start integrating your security incident and response. 
+title: Turn on Microsoft Threat Protection - Microsoft 365 security
+description: Learn how to enable Microsoft Threat Protection (MTP) and start integrating your security incident and response. 
```

Comment:
Are we allowed to use "MTP" on public-facing text? The description will show up on SERP.